kik_unofficial/client.py

```diff
@@ -913,10 +913,18 @@ def is_multi_packet(self, data: bytes):
     def cleanup_buffer(self):
         # This method is called after the cleanup_interval to remove outdated data from the buffer
         current_time = time.time()
+        cleaned_count = 0  # Initialize a counter for cleaned messages
+
         self.logger.debug("Cleaning up partial data buffer")
         self.data_buffer = [(ts, data) for (ts, data) in self.data_buffer if current_time - ts <= self.cleanup_interval]
+
```

Comment:
If sounds like `data_buffer` is a buffer, but in fact it's an array of timestamps and buffers. Let's make this clear

Reply:
Changed to data_array